README.md

```diff
@@ -289,9 +289,15 @@ $LLVM_DIR/bin/clang -cc1 -load <build_dir>/lib/libLACommenter.dylib -plugin LAC
 but without the need of using `clang` and loading the plugin:
 
 ```bash
-<build_dir>/bin/ct-la-commenter input_file.cpp
+<build_dir>/bin/ct-la-commenter input_file.cpp --
 ```
 
+`--` appended at the end to prevent the complain of clang tools about missing 
+compilation database. Another workaround would be here is to set the 
+CMAKE_EXPORT_COMPILE_COMMANDS flag during the cmake invocation. It will give 
```

Comment:
[nit] Small suggestion:  perhaps add a link to https://cmake.org/cmake/help/v3.14/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html?

Reply:
Ok